presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,75 @@
+=============
+Release 0.273
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator resource group aggregation issue related to WAITING_FOR_PREREQUISITE queries.
+* Fix Iceberg $files table in case of column dropping.
+* Fix a bug where ``EXPLAIN (TYPE IO)`` did not include the index source from an index join.
+* Fix a memory overcounting issue for using varchar types that could cause queries to fail with exceeded memory limit errors.
+* Fix an error in ``tdigest_agg``.
+* Add `to_base32` and `from_base32` varbinary functions.
```

Comment:
Functions have to be written with `:func:`
See [this](https://github.com/prestodb/presto/commit/e4dd2048b38e2677792415bb561e639cb05bd9eb#diff-31566b652020a06db29e535cd1943a102fb3a44b568963879ff1e05130a5fb96R7) as an example.

Reply:
done